app/assets/javascripts/admin-conference.js

```diff
@@ -325,6 +337,25 @@ require(["lib/models", "lib/tools", "lib/owned", "knockout", "ko.sortable", "dat
                 return;
             }
 
+            var imageUrlsChange = Array();
+            if (self.logoURL() !== null || self.logoURL() !== "") {
+                imageUrlsChange.push("logo: " + self.logoURL());
+            }
+            if (self.thumbnailURL() !== null || self.thumbnailURL() !== "") {
+                imageUrlsChange.push("thumbnail: " + self.thumbnailURL());
+            }
+            if (imageUrlsChange.length > 0) {
+                imageUrlsChange.forEach(function (value) {
+                    var iuType = value.split(":")[0];
+                    for (var i = 0; i < self.conference().imageUrls().length; i++) {
+                        if (self.conference().imageUrls()[i].search(iuType) >= 0) {
+                            self.conference().imageUrls().splice(i);
+                        }
+                    }
+                    self.conference().imageUrls().push(value);
+                });
+            }
```

Comment:
This block seems like an overly complicated construction. Please check, if a push of new or direct replacement of existing entries in `self.conference().imageUrls()` is possible without the use of the `imageUrlsChange` variable.